### PR TITLE
Feature/email module

### LINKED ISSUE #52: Send emails with password when users is create
- [x] Setup sendgrid

- [x] Implementation

- [ ] Test send emails

### PR DESCRIPTION
**Pull request by jcmexdev**

### CODE REVIEW

File: src/modules/user/user.service.ts [hunk 1]
```diff
@@ -37,14 +44,22 @@ export class UserService {
...
+
+    await theUser.save()
+
+    const fullName = `${theUser.firstname} ${theUser.lastname}`;
+
+    await this._sendGrid.send({
+      to: theUser.email,
+      from: new ConfigService().get(ConfigEnum.BUSINESS_MAIL),
+      subject: "Welcome to Fast Order",
+      html: emailTemplate(fullName, theUser.email, password),
```
Comment (jlazo) [inline src/modules/user/user.service.ts]: Aquí, creo que sería mejor un object como parametro, para no tener que enviar tantos

**Review by jlazo — COMMENTED** [review]
The yarn.lock is not necessary upload

File: src/modules/user/user.service.ts (hunk 1 above)
Comment (jcmexdev) [reply src/modules/user/user.service.ts]: i get it, i'm going to remove it

**Review by jlazo — APPROVED** [review]

File: src/modules/utils/email-welcome-template.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+          <td style="text-align: center">
+            <a
+              style="
+                font-family: Helvetica;
+                text-shadow: 1px 3px #525252;
+                color: #f1f7fe;
+                text-decoration: none;
+              "
+              href="https://www.google.com/"
+              >Go to Login</a
```
Comment (AlanDavd) [inline src/modules/utils/email-welcome-template.ts:73]: Why log in is linked to Google?

File: src/modules/user/user.service.ts [hunk 3]
```diff
@@ -37,14 +43,26 @@ export class UserService {
...
-      password,
-    );
+
     const salt = await genSalt(10);
     theUser.password = await hash(password, salt);
-    return plainToClass(UserResponseDto, await theUser.save());
+
+    await theUser.save();
+
+    let emailInfo = {
```
Comment (AlanDavd) [inline src/modules/user/user.service.ts]: Just a semantic change, emailInfo is never re-assign, we should use const instead of let

**Review by AlanDavd — APPROVED** [review]
LGTM

**Pull request merged**